src/femr/labelers/omop.py

```diff
@@ -269,7 +396,7 @@ def __init__(
         outcome_codes: List[str],
         time_horizon: TimeHorizon,
         prediction_codes: Optional[List[str]] = None,
-        prediction_time_adjustment_func: Callable = identity,
+        prediction_time_adjustment_func: Optional[Callable] = None,
```

Comment:
Any particular reason from going from the old identity to None values here?

Reply:
Yeah the default `multiprocessing` module won't let you use `identity` as a default function argument since its a lambda function -- that's why I originally had included the `pathos` library since it supports sending lambda functions as default arguments to processes. 

Setting the default to `None` avoids the issue with `multiprocessing`